javageci-core/src/main/java/javax0/geci/equals/Equals.java

```diff
@@ -88,11 +89,18 @@ public void postprocess(Source source, Class<?> klass, CompoundParams global) th
     private void generateEqualsHeader(Segment segment, Class<?> klass, CompoundParams global) {
         var equalsMethod = getMethodOrNull(klass, "equals", Object.class);
         var subclassingAllowed = global.is("subclass", config.subclass);
+        var usingSuper = global.is("useSuper", config.useSuper);
```

Comment:
Here we rather should create a `final var local = localConfig(global);` and then use `local.useSuper` which simply implements the same logic but for all the configuration options.